release.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python
```

Comment:
I don't know if everyone would agree with me, but this could be inside a `.ci` directory of some kind, specially if ignored by the `MANIFEST.in` file. Looking at the `MANIFEST.in` file now, I actually see that we have some work to do in there as well ...

Reply:
@vltr I like the idea of keeping all CI/CD tools in a common directory so that they are out of the way of regular development. 

As for the `MANIFEST.in` goes, I was thinking of doing it as a different PR. But I can change it as part of this one as well if you want me to. 

Just an FYI, check out https://flit.readthedocs.io/en/latest/

Reply:
@harshanarayana yeah, there are some tools today that can do that for us ... There is `towncrier` to automate the changelog entries (you can [see it in use](https://github.com/python-attrs/attrs/blob/2b31a30f87599c79df42ed5353a3d8c274e0e1cd/pyproject.toml) in the `attrs` project). There's `poetry` that can handle some of the same stuff that `flit` does, but I actually have never used them and, while the [_eternal discusion_](https://np.reddit.com/r/Python/comments/8jd6aq/why_is_pipenv_the_recommended_packaging_tool_by/) over a new tool that "does everything in packaging" isn't over with a consensus, I guess we'll stick to `setuptools` for a while ...

Reply:
```bash
➜ check-manifest -u -v
listing source files under version control: 185 files and directories
building an sdist: sanic-0.8.3.tar.gz: 185 files and directories
copying source files to a temporary directory
building a clean sdist: sanic-0.8.3.tar.gz: 185 files and directories
lists of files in version control and sdist match
```

Manifest Coverage after new changes to manifest

Reply:
Wow! Are you trying to do all the hardwork by yourself? :sweat_smile:

Indeed, `towncrier` seems like a nice tool to provide a complete changelog and that kind of stuff, even though I have not used it myself (yet).

As for the `MANIFEST.in` file, I think that the wheel distribution should have as little as possible (IMHO). I would exclude almost everything except for code, `LICENSE` and the `setup.py`. But that's just my opinion. I don't know if we can make different packages for `wheel` and `sdist`. For `sdist` I would just add everything that's not related to our development process, leaving mostly files like `setup.cfg` (it just holds information for dev as well), `tox.ini`, `.travis.yml` and etc out.

Reply:
That's interesting. I would expect the `whl` file to have the minimum contents, I just wanted to make sure it doesn't includes examples and tests (all containing `.py` files that, well, might get inside for some reason unknown). Now, the `sdist` I think would require the minimum to execute `python setup.py install` and that would be it :wink: